tests/test_webhook.py

```diff
@@ -177,6 +180,17 @@ def test_extract_priority_label(self):
         priority_label_none = helpers.extract_priority_label(self.issue_body)
         self.assertEqual(priority_label_none, None)
 
+    def test_get_issue_labels(self):
+        """Extract list of labels from an issue body."""
+        labels_tests = [
+            (self.issue_body, ['browser-firefox', 'type-media', 'type-stylo']),
+            (self.issue_body2, ['type-foobar']),
+            (self.issue_body3, ['browser-firefox-mobile-tablet'])
+        ]
+        for issue_body, expected in labels_tests:
+            actual = helpers.get_issue_labels(issue_body)
+            self.assertEqual(expected, actual)
```

Comment:
Here you are always testing a list. So we could use.

`self.assertListEqual(expected, actual)` but this would fail if the list items have a different order. 

`self.assertEqual(sorted(expected), sorted(actual))` will fix that. Then we have a more robust test.